src/components/EntityInfo/EntityMenu.vue

```diff
@@ -163,6 +163,48 @@
               </template>
               Submit a Consent to Continue Out of the province of B.C.
             </v-tooltip>
+
+            <!-- Request AGM Extension -->
+            <v-tooltip
+              right
+              content-class="right-tooltip"
+            >
+              <template #activator="{ on }">
+                <v-list-item
+                  v-if="enableAgmExtension"
```

Comment:
Only show this menu item if the feature is enabled via FF.

Oddly, this didn't work when I put the FF check here directly, so I put it in a getter (computed) instead.

Reply:
I have experienced something similar (do not remember exactly which code base) but works in a getter

Reply:
Actually, this worked when running locally, but not in the unit tests 🤷‍♂️